test/no-functions.js

```diff
@@ -5,13 +5,17 @@ var Understudy = require('../').Understudy;
 var actor = new Understudy();
 
 actor.before('no-functions', function () {
-  assert.equal(true, false);
+  throw new Error('before shouldn\'t be reached');
 });
 
-actor.after('no-functions', function () {
```

Comment:
It's not redundant. It's an additional assertion that should never be hit: after hooks should not be executed. If they are it's a bug.